ddtrace/propagation/http.py

```diff
@@ -611,12 +612,11 @@ def _get_traceparent_values(tp):
         return trace_id, span_id, sampling_priority
 
     @staticmethod
-    def _get_tracestate_values(ts):
-        # type: (str) -> Tuple[Optional[int], Dict[str, str], Optional[str]]
+    def _get_tracestate_values(ts_l):
+        # type: (List[str]) -> Tuple[Optional[int], Dict[str, str], Optional[str]]
 
-        # tracestate parsing, example: dd=s:2;o:rum;t.dm:-4;t.usr.id:baz64,congo=t61rcWkgMzE
+        # tracestate list parsing, example: [dd=s:2;o:rum;t.dm:-4;t.usr.id:baz64,congo=t61rcWkgMzE]
```

Comment:
Is this an example of an input? Isn't the input supposed to be a list of strings?

Reply:
ah you're right, need to add the quotes. Good call.